app/presenters/admin_auction_status_presenter/available.rb

```diff
@@ -4,7 +4,26 @@ def header
   end
 
   def body
-    I18n.t('statuses.bid_status_presenter.available.admin.body', end_date: end_date)
+    if total_bids > 0
```

Comment:
I would probably default to doing this check in out big giant conditional rather than here...but I can see why we'd want it here. I think that we need to work on breaking up that giant `if` statement so it is easier to read and understand. So having this here for now is easier to comprehend. In any ideal world, I think we'd keep all our conditionals outside of presenters